main.py

```diff
@@ -93,3 +93,16 @@ def add_nosniff_content_type_header(response):
     response.headers["X-Content-Type-Options"] = "nosniff"
     response.headers["X-Frame-Options"] = "deny"
     return response
+
+@app.after_request
+def add_strict_transport_security_header(response):
+    # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
+    # recommended settings by mozilla, max-age is 2 years in seconds
```

Comment:
I took a look through the link (which links to hstspreload.org) and there's something about:
> Once you're confident that there will be no more issues, increase the max-age to 2 years and submit your site to the preload list

from https://hstspreload.org/

I'm not very familiar with the preload list but is it something we should be concerned about?

Reply:
@andy7i I think this is more of a devops thing where they would register the terra ui dns record with that hstspreload list

Reply:
it seems to be out of scope of this ticket and pentest requirements

Reply:
one of my concerns was that 2 years might not be the right amount of time to put here. Mozilla says:

> **If** a max-age of 1 year is acceptable for a domain, however, two years is the recommended value as explained on https://hstspreload.org/.

emphasis on `if`. Do we know if 1 year is acceptable?

if you look at https://hstspreload.org/ it has deployment recommendations that slowly ramp up the max-age (starting with 5 minutes and 1 week intervals) .